apps/fluent-tester/src/FluentTester/TestComponents/FocusZone/FocusZoneTest.tsx

```diff
@@ -45,7 +43,28 @@ const EdgeCasesFocusZone: React.FunctionComponent = () => {
       <FocusZone>
         <SubheaderText>FocusZone with no elements</SubheaderText>
       </FocusZone>
-    </View>
+      <SubheaderText>Nested FocusZones</SubheaderText>
+      <Button>Outside Focus Zone</Button>
+      <Text>Parent FocusZone</Text>
+      <View style={focusZoneTestStyles.nestedFocusZoneStyle}>
+        <FocusZone>
+          <Text>Inner FocusZone 1</Text>
+          <View style={focusZoneTestStyles.nestedFocusZoneStyle}>
+            <FocusZone>
+              <ListOfCheckboxes />
+            </FocusZone>
+          </View>
+          <Text>Inner FocusZone 2</Text>
+          <View style={focusZoneTestStyles.nestedFocusZoneStyle}>
+            <FocusZone>
+              <ListOfCheckboxes />
+            </FocusZone>
+          </View>
+          <Button>Inside Focus Zone</Button>
+        </FocusZone>
+      </View>
+      <Button>Outside Focus Zone</Button>
```

Comment:
super nit: also padding around the button as well for aesthetics and enough space for blue ring to show up?